app/views/new_filters/_location_provider_filter.html.erb

```diff
@@ -0,0 +1,30 @@
+<div class="app-filter__group" data-qa="filters__area_and_provider">
+  <legend class="govuk-fieldset__legend govuk-fieldset__legend--s">
+    <h3 class="govuk-fieldset__heading">
+      <% if @filters_view.provider_query? %>
+        Provider
+      <% else %>
+        Location
```

Comment:
This says 'Area' in the prototype. I think location might make more sense to use for now though. Happy to defer to Paul though.

Reply:
Yep, we’ll use ‘Location’ for now, can change it to ‘Area’ when we’re using the new location… area model.

Reply:
Should have checked your comments 🤦